hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -698,6 +701,12 @@ public boolean mkdirs(Path f, FsPermission permission) throws IOException {
     return mkdir(f);
   }
 
+  @Override
+  public long getDefaultBlockSize() {
+    return (long) getConf().getStorageSize(
+        OZONE_SCM_BLOCK_SIZE, OZONE_SCM_BLOCK_SIZE_DEFAULT, StorageUnit.BYTES);
```

Comment:
What if the client who wrote the block and the client who is querying have a different configuration?

Reply:
This is intended to be used by the client that creates the file.  Once the file is closed, the actual block size can be queried via `getBlockSize(Path)`.  Currently this method returns 32MB, regardless of SCM config:

https://github.com/apache/hadoop/blob/a6df05bf5e24d04852a35b096c44e79f843f4776/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystem.java#L2521-L2522

Reply:
Thanks @arp7 for taking a look.  Does the above answer your quetion, or would you like further clarification?

Reply:
Yes, thanks. Sorry i missed responding.